addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -1082,11 +1083,7 @@ def test_17_deliver_more_and_multi_uom(self):
         self.assertEqual(so.order_line[1].product_uom.id, uom_km_id)
 
 
-class TestSaleStockOnly(TestSaleCommon):
```

Comment:
I suppose that the same can be done in `procurement_jit`, lets save us the complexity of the setup of `TestSaleCommon`

Reply:
Done, thank you